cmd/zqd/listen/command.go

```diff
@@ -45,7 +47,8 @@ func (c *Command) Run(args []string) error {
 		return err
 	}
 	logger := newLogger()
-	handler := zqd.NewHandler(dataDir)
+	core := zqd.Core{Root: dataDir, ZeekExec: c.zeekExec}
+	handler := zqd.NewHandler(core)
```

Comment:
I like the idea of a `Core` struct. I think it should be passed as a pointer-to-struct parameter instead of a struct parameter in the all of the places it's used.

Reply:
Good idea.